source/leuven/credits.rst

```diff
@@ -48,52 +44,130 @@ Please contact your VSC coordinator/contact or your :ref:`local support staff
-the walltime it consumed, and the number and type of compute nodes it
-ran on. The following formula is used:
+On Tier-2 clusters, we use Slurm for accounting purposes (on top of resource and 
+job management).
+See :ref:`Slurm accounting <accounting_leuven>` page for additional information.
+In Slurm terminology, the cost of a job depends on the trackable resources (TRES)
+it consumes. Two distinct TRES are the number of CPU cores and GPU devices. 
+Different types of CPU and GPU nodes are given different weights 
+(``TRESBillingWeights``) which you can retrieve as follows for e.g. wICE::
+
+   scontrol show partitions -M wice
+
+As an example, for a GPU node, the weights are configured as::
+
+   TRESBillingWeights=CPU=2.546296296,GRES/gpu:a100-sxm4-80gb=141.6666667,GRES/shard:a100-sxm4-80gb=141.6666667
+
+**CPU-only jobs**
+
+The following formula applies::
+
+   CPU TRESBillingWeights * num_cores * walltime
+
+Where
+
+- ``CPU TRESBillingWeights`` is the applied weight for CPU resources (see above)
+- ``num_cores`` is the *effective* number of cores used for the job
+- ``walltime`` is the number of minutes that the job ran
+
+**GPU jobs**
 
-0.000278 \* *nodes* \* *walltime* \* *nodetype*
+The following formula applies::
 
+   (CPU TRESBillingWeights * num_cores + GPU TRESBillingWeights * num_gpus) * walltime
+   
 Where
 
--  *nodes* is the number of compute nodes the job ran on;
--  *walltime* the effective duration of the job, expressed in seconds;
--  *nodetype* is the factor representing the node type's performance as
-   listed in the table below.
+- ``CPU TRESBillingWeights`` is the applied weight for CPU resources (see above)
+- ``GPU TRESBillingWeights`` is the applied weight for GPU resources (see above)
+- ``num_cores`` is the *effective* number of cores used for the job
+- ``num_gpus`` is the *effective* number of GPUs used for the job
+- ``walltime`` is the number of minutes that the job ran
 
-The Tier-2 cluster has several types of compute nodes, none of which
-is actually a reference node. The values for the different types are
-listed at the following links:
+.. note::
 
-- :ref:`Genius <running jobs on genius>`
+    *Effective* number of cores is not necessarily equal to what the user requests.
+    E.g. if a job requests a single core/task, but the full memory of a node on wICE,
+    then one node is blocked for such a job. Consequently, the effective number of cores
+    will be 72, instead of 1.
+
+    Similarly, on a node with 4 GPUs, if a user requests to use a single GPU, one-fourth
+    of cores will be assigned to the job. So, e.g. on wICE ``num_cores`` will be 18.
+
+.. note::
+
+    *Effective* number of GPUs deserves an explanation. If a user requests e.g. 2 GPUs
+    for a job, then ``num_gpus`` will be 2. However, in certain cases, we allow multiple
+    users to share a GPU (called sharding). In such cases, ``num_gpus`` could be a factor
+    between zero and one.
```

Comment:
```suggestion
```
We have not (yet) implemented a mechanism in our Slurm setup that would ensure that a certain cores-to-GPU ratio is applied in GPU jobs (e.g. 18:1 in the case of wICE). So it is best to just describe it as it is (i.e. without the notion of "effective" number of GPUs). The description of sharding is also not correct/complete here, and I suggest to just leave it out (at least for now), since this is only supposed to be used for specific courses, and definitely not in general.

Reply:
fair enough. I adapted the text, and also removed the corresponding "note" that elaborated the "effective gpus".